lib/shopify-cli/messages/messages.rb

```diff
@@ -334,7 +334,7 @@ module Messages
           start: "{{v}} ngrok tunnel running at {{underline:%s}}",
           start_with_account: "{{v}} ngrok tunnel running at {{underline:%s}}, with account %s",
           stopped: "{{green:x}} ngrok tunnel stopped",
-          timed_out: "{{x}} ngrok tunnel has timed out, restarting ...",
+          timed_out: "{{x}} ngrok tunnel has timed out, restarting …",
```

Comment:
Can we drop the space before the `…`? (I realise it was already there, but seems more noticeable now).

Reply:
@andyw8 Done, and I checked, there isn't any other similar issue.